File: employee-service/src/main/java/com/okhanzhin/employee/service/model/Workstation.java
package com.okhanzhin.employee.service.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Workstation {

    /**
     * Идентификатор рабочего места
     */
    private Long id;

    /**
     * Название рабочего места
     */
    private String name;

    /**
     * Операционная система рабочего места
     */
    private String os;
}

File: workstation-service/src/main/kotlin/com/okhanzhin/workstation/service/model/Workstation.kt
package com.okhanzhin.workstation.service.model

data class Workstation(

	/**
     * Идентификатор рабочей станции
     **/
	val id: Long,

	/**
     * Название рабочей станции
     */
	val name: String,

	/**
     * Операционная система рабочей станции
     */
	val os: String
)

